http-proxy/conf.d/ngnix.conf

```diff
@@ -2,6 +2,8 @@
 server {
 
     listen 80;
+    client_max_body_size 8M;
+    add_header Access-Control-Allow-Origin "localhost:3000";
```

Comment:
Please explain the situation why this is needed? A comment could be added in code to help.

If needed, conf.d-prod/nginx.conf also needs to be updated, or this will have no effect on the compiled image.